packages/banner/src/BannerPresenter/presenters.js

```diff
@@ -142,3 +231,63 @@ export function InteractionsWrapper({ innerRef, children }) {
     </ThemeContext.Consumer>
   );
 }
+
+/** @type {Object.<string, string>} */
```

Comment:
I think this block of `const` should go higher in the file after the import and before the function exports as they may make more sense to be declared up top.

The other thing you could consider is since `iconNamesByType` seems to be only used by the `IconBackground` function, you can declare these `const` from within the function instead.